### DISCUSSION TITLE
Object Address always empty for Handles ?

### DISCUSSION THREAD

**Discussion by UnrealKaraulov** [winsiderss/systeminformer#2227] [Questions / Answers] [answered]
Object Address always empty for Handles in SystemInformer ?

But for example using blackbone I can get list of handles with object addresses but all is invalid :(

**Comment by ge0rdi** [comment]
You need to enable kernel driver (`Options` -> `Enable kernel-mode driver`) in order to get all the information.

**Comment by dmex** [comment, marked as answer]
The object addresses are zero because the handle snapshot feature included with Windows doesn't include the object addresses in snapshots.

You can disable handle snapshots by changing the `EnableHandleSnapshot`  setting to `0` but Windows will be a slower enumerating handles.
